cli/pcluster/pcluster.py

```diff
@@ -577,7 +577,36 @@ def instances(args):
         LOGGER.info("Run 'awsbhosts --cluster %s' to list the compute instances", args.cluster_name)
 
 
-def command(args, extra_args):
+def get_master_server_ip(stack_name, config):
+    ec2 = boto3.client(
+        "ec2",
+        region_name=config.region,
+        aws_access_key_id=config.aws_access_key_id,
+        aws_secret_access_key=config.aws_secret_access_key,
+    )
+
+    master_id = get_master_server_id(stack_name, config)
+    if master_id is []:
+        LOGGER.info("MasterServer not running. Can't SSH")
+        sys.exit(1)
+    instance = ec2.describe_instances(InstanceIds=[master_id]).get("Reservations")[0].get("Instances")[0]
+    ip_address = instance.get("PublicIpAddress")
```

Comment:
Should we add the fallback logic to the PrivateIpAddress here too?

Reply:
added